documentation/BlockConstruct.md

```diff
@@ -0,0 +1,53 @@
+<!--
+Copyright (c) 2019, Arm Ltd.  All rights reserved.
+-->
+# Design proposal for Block Construct statement semantic analysis
+# The implementation of Block construct is based on the Section 11.1.4 of
+# Fortran 2018 standards Document
+
+The semantic checks of Block Construct in Section 11.1.4 are going to be
+implemented in these new files:
+ check-block.cc and check-block.h
+
+The new files check-block.•   implements/have BlockChecker class.
```

Comment:
Something is messed up on this line after `check-block.`

Reply:
It should be check-block.*